binaryaudit/dnf.py

```diff
@@ -45,8 +46,7 @@ def process_downloads(source_dir, new_json_file, old_json_file, output_dir, buil
             ret_status = generate_abidiffs(key, source_dir, new_json_file, old_json_file, output_dir,
                                            conf_dir, build_id, product_id, db_conn, cleanup)
             util.note("Status: " + ret_status)
-
-            if ret_status == "fail":
+            if ret_status == "INCOMPATIBLE_CHANGE":
```

Comment:
Not related to this patch directly, but i'd suggest indeed target returning the numeric value of the exit code in some follow up work. Then it can be evaluated outside of this functions, and otherwise would be as simple as `status != 0` is a failure.

Thanks.